config/importConfig.importorder

```diff
@@ -0,0 +1,6 @@
+#Organize Import Order
+#Thu Jun 04 03:11:49 IST 2020
+0=java
+1=javax
+2=org
+3=com
```

Comment:
Nope, we do NOT want this, agreed? See longer comment in main discussion thread.

Reply:
yes!

Reply:
done (replaced by `importOrder() //sort imports alphabetically`)